recipes/dysgu/meta.yaml

```
@@ -24,8 +24,7 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - python >=3.7
-    - setuptools >=63.0
+    - python
```

```suggestion
    - python
    - setuptools >=63.0
```
setuptools is still needed: 
```
"ERROR: Can not execute `setup.py` since setuptools is not available in "[0m
2024-07-02T19:34:33.4200343Z 19:34:33 [32mBIOCONDA INFO[0m (OUT) [34m          "the build environment.",[0m
```